4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -5,19 +5,19 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 # Install NuGet CLI
 RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; \
     Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v4.3.0/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe; \
-    setx /M PATH $($Env:PATH + ';' + $Env:ProgramFiles + '\NuGet')
+    & setx /M PATH $($Env:PATH + ';' + $Env:ProgramFiles + '\NuGet')
```

Comment:
Why setx over `ENV` directive in the Dockerfile? I thought setx won't won't be inherited if another Dockerfile (aspnet-build) depends on this?

Reply:
`ENV` does not support path concatenation on Windows - see https://github.com/moby/moby/issues/22017.  My tests on with the aspnet-build images indicate this works as expected.  I will do some more testing though.